index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+        <figure id="img-div">
+          <img
+            id="image"
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+            alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
+          />
+          <figcaption id="img-caption">
+            Dr. Norman Borlaug, third from the left, trains biologists in Mexico on
+            how to increase wheat yields - part of his life-long war on hunger.
+          </figcaption>
+        </figure>
+        <section id="tribute-info">
+          <h3 id="headline">Here's a time line of Dr. Borlaug's life:</h3>

```

Comment:
You should have to wrap all the ul tags in the article. By this way your code will look more professional. :+1: